src/css/global.css

```
@@ -68,9 +74,27 @@ header {
     width: 100%;
 }
 
+.center {
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    margin: auto;
```

bruh what ? :moyai: :moyai: 
```suggestion
    top: 50%;
    left: 50%;
```